entrypoint.sh

```diff
@@ -56,5 +56,7 @@ pip install -r requirements.txt
 # Download the latest SSN Console Tools if necessary
 if [ ! -d "solsta_console" ]; then python3 direct_get.py --overwrite --version="$INPUT_CONSOLE_VERSION" --target_directory=./solsta_console/ --console_credentials=client_credentials.json ; fi
 # Run the script that creates a new release and deploys it
+solsta_extra_args=""
+if [ ! "$INPUT_RELEASE_VERSION" = "" ]; then solsta_extra_args="--version=\"$INPUT_RELEASE_VERSION\"" ; fi ;
```

Comment:
Unless there's plans to expand this, I'd go with something a little more specific (e.g. solsta_version_args).

Reply:
If you are planning on expanding, then I suggest using solsta_extra_args += " --version..." (note the prepended space)